src/nih_wayfinding/app/index.html

```diff
@@ -101,6 +102,7 @@ <h2 class="navbar-actionheader" ng-cloak ui-sref="profiles">
     <!-- endbuild -->
 
     <!-- build:js({.tmp,app}) scripts/scripts.js -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Leaflet.awesome-markers/2.0.2/leaflet.awesome-markers.js"></script>
```

Comment:
Looks like it has a [bower.json](https://github.com/lvoogdt/Leaflet.awesome-markers/blob/2.0/develop/bower.json).